projects/smoke_bts7200/src/current_measurement_config.c

```diff
@@ -0,0 +1,212 @@
+#include "current_measurement_config.h"
+#include "pin_defs.h"
+
+// Definitions of the hardware configs declared in the header
+
+#define POWER_DISTRIBUTION_I2C_PORT I2C_PORT_2
+
+const PowerDistributionCurrentHardwareConfig FRONT_POWER_DISTRIBUTION_CURRENT_HW_CONFIG = {
+  .i2c_port = POWER_DISTRIBUTION_I2C_PORT,
+  .dsel_i2c_addresses =
+      (I2CAddress[]){
+          POWER_DISTRIBUTION_I2C_ADDRESS_0,  //
+          POWER_DISTRIBUTION_I2C_ADDRESS_1,  //
+      },
+  .num_dsel_i2c_addresses = 2,
+  .bts7200s =
+      (PowerDistributionBts7200Data[]){
+          {
+              .dsel_pin = FRONT_PIN_LEFT_RIGHT_CAM_DSEL,
+              .current_0 = FRONT_POWER_DISTRIBUTION_CURRENT_LEFT_CAMERA,
+              .current_1 = FRONT_POWER_DISTRIBUTION_CURRENT_RIGHT_CAMERA,
```

Comment:
Like I commented above, I think you can remove these from each entry in this file. This will cut down on the unnecessary code.